CONTRIBUTING.md

```diff
@@ -1,9 +1,18 @@
 ## Contributing
-GeoBlacklight is a collaborative open source projects where contributions are :sparkles:welcome:sparkles:. This contributing guide is borrowed in part from the [Samvera Contributing Guide](https://github.com/samvera/hydra/blob/master/CONTRIBUTING.md) and the [Blacklight Contributing Wiki](https://github.com/projectblacklight/blacklight/wiki/Contributing-to-Blacklight).
+GeoBlacklight is a collaborative, open source project where contributions are :sparkles:welcome:sparkles:. This contributing guide is borrowed in part from the [Samvera Contributing Guide](https://github.com/samvera/hydra/blob/master/CONTRIBUTING.md) and the [Blacklight Contributing Wiki](https://github.com/projectblacklight/blacklight/wiki/Contributing-to-Blacklight).
```

Comment:
Per the group discussion today, we might cut the reference to Samvera and Blacklight. Also, in the website/documentation contribution guide, I added a link to this guide - do we want to do the same to cross-link? Something like:

```
To contribute to our website or documentation pages, see the [GeoBlacklight Website Contribution Guide](https://github.com/geoblacklight/geoblacklight.github.io/blob/main/CONTRIBUTING.md).
```

Reply:
Putting this down with _Contributing code or documentation_